brave/brave-core
# Create P3A utils + add Wallet new user p3a metric

Resolves brave/brave-browser#23463

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

0) Start browser with fresh profile
1) Access local state page, verify that the metric exists in the p3a object and has a value of 0
2) Create Wallet
3) Verify that the metric now exists and has a value of 2
4) Close browser, fast forward the time by one day, open browser, access wallet
5) Verify that the metric has a value of 3
6) Close browser, fast forward the time by one week, open browser, access wallet
7) Verify that the metric has a value of 1
